src/dgraph/dgraph.service.ts

```diff
@@ -180,11 +180,14 @@ export class DgraphService implements OnModuleInit {
       type DIDDocument {
         id
         logs
+        owner
+        offeredTo
```

Comment:
IMHO, it would be nice if the Asset/OfferableIdentity data was grouped/separate from DIDDocument to some extent. Like maybe an Asset inherits from DIDDocument somehow. Or maybe a nice approach is to have an Asset type that has both a DIDDocument and an OfferableIdentity? Something like this?
```
type Asset {
  document DIDDocument
  ownership OfferableIdentity
}
type OfferableIdentity {
  owner
  owfferTo
}
```

Reply:
This is why I hate dgraph, cause if we want to do that we'll need to prevent of creating new node's on every update (OfferableIndentity is a separate node)